### ISSUE TITLE
Modifying Webpack Configurations error

### ISSUE THREAD

**Issue by mkim871** [electron-userland/electron-webpack-quick-start#41]
I have followed below tutorial to modify webpack configuration but this occurs an error.
https://webpack.electron.build/modifying-webpack-configurations

```
  Module build failed (from ./node_modules/babel-loader/lib/index.js):
  Error: [BABEL] D:\workspace\5-Seaters-Desktop\src\renderer\index.js: .default is not a valid Plugin property
```

I have not changed any other codes and also tried solution from follwing link.
https://stackoverflow.com/questions/52109671/babel-default-is-not-a-valid-plugin-property-at-object-keys-foreach-key/52110352#52110352

**Comment by christophpre** [comment]
electron-webpack-quick-start currently uses a version of electron-webpack **2.1.2**, that doesn't contain the fix for the new babel version. Have you updated electron-webpack in your project to **2.3.1**?

**Comment by mkim871** [comment]
Yes I am currently using 2.3.1

**Comment by mkim871** [comment]
Yes I have solved this issue by updating it to 2.3.1
